cms/templates/js/course-outline.underscore

```diff
@@ -177,6 +177,17 @@ if (is_proctored_exam) {
                         <% } %>
                     </p>
                 </div>
+                <div class="status-hide-after-due">
+                    <p>
+                        <% if (xblockInfo.get('hide_after_due')) { %>
+                            <i class="icon fa fa-eye-slash"></i>
+                            <span class="status-hide-after-due-value"> <%= gettext("Exam will remain hidden after due date") %> </span>
```

Comment:
Should be <%- to be XSS safe (in case of nefarious translation, and our best practice is to always escape unless it is known to be HTML).

Same below.